.github/workflows/ci.yml

```diff
@@ -58,9 +58,32 @@ jobs:
       - name: Run code style check
         run: yarn style
 
+  setup-common-test:
+    runs-on: ubuntu-latest
+    outputs:
+      common-tests: ${{ steps['set-common-tests'].outputs['common-tests'] }}
+      set-common-test-names: ${{ steps['set-common-test-names'].outputs['common-test-names'] }}
+      registry-image-tag: ${{ steps['set-registry-image-tag'].outputs['registry-image-tag'] }}
```

Comment:
Is this going to work? I think there's no `registry-image-tag` step in this job. Or maybe it's failing silently since we're not using that variable anywhere.

Reply:
I think this is not used here

Reply:
Oh, crap, now I see what you're referring to. Oops. :(